src/sdk/util/request.ts

```diff
@@ -1,12 +1,14 @@
 import { PublicKey } from '@wharfkit/antelope';
 import { VCWithTypeType, VerifiableCredentialOptions, VerifiableCredentialWithType } from './ssi/vc';
 import { Issuer } from '@tonomy/did-jwt-vc';
+import { DataSharingRequest } from './dataSharingRequest';
 
 export type LoginRequestPayload = {
     randomString: string;
     origin: string;
     publicKey: PublicKey;
     callbackPath: string;
+    dataSharingRequest?: DataSharingRequest;
```

Comment:
This should not be here, instead we should have it as part of `LoginRequestsMessagePayload,`

https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/blob/master/src/sdk/services/communication/message.ts#L118C1-L120C3
```ts
export type LoginRequestsMessagePayload = {
    requests: Request[];
};
```